eos_data_distribution/ndn/file.py

```diff
@@ -82,25 +83,32 @@ def mkdir_p(dirname):
 SEGMENT_TABLE_MAGIC = 'EosSgtV1'
 
 # 7 => [0, 0, 0, 0, 0, 1, 1, 1]
+
+
```

Comment:
I think a better fix here would be to put the comment text in a docstring for the function, rather than effectively moving it further away from the function it documents.